CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 Changes in 0.8.1
 ----------------
 - Added Django 1.10 support `198 <https://github.com/django-money/django-money/issues/198>`_ (`Dmitry Dygalo <https://github.com/Stranger6667>`_)
+- Changed auto conversion of currencies using djmoney_rates (added in 0.7.3) to
```

Comment:
I didn't do anything on this yet - are you saying we don't need a version bump? The latest released version is 0.8 right?

Also are you OK with the code duplication above for the reasons given?